### PR TITLE
Use our own docker image, cleanup contrib docs

### PR DESCRIPTION
**Pull request by tboerger**
* Use our own docker image, cleanup while reading
* Add prose and serve to package with yarn tasks

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -127,4 +127,4 @@ pdf-user:
 .PHONY: check-prose
 check-prose:
 	@echo "Checking quality of the prose in all files"
-	write-good --parse **/*.adoc
+	@yarn prose
```
Comment (mmattel) [inline Makefile]: How does yarn prose know which files to scan ?

File: docs/build-the-docs.md [hunk 2]
```diff
@@ -20,29 +20,28 @@ docker run -ti --rm \
...
 ```
 
 This command:
 
-- Starts up [ownCloud's custom Antora Docker container](https://hub.docker.com/r/antora/antora/)
-- Removes any existing documentation
+- Starts up [ownCloud's custom Antora Docker container](https://hub.docker.com/r/owncloudci/antora/)
 - Runs Antora's `generate` command, which regenerates the documentation
-- You can add the `--clean` option to clean the build directory of any leftover artifacts from the previous build, including pdf's.
+- You can add the `--clean` option to clean the build directory of any leftover artifacts from the previous build, including ODF's.
```
Comment (mmattel) [inline docs/build-the-docs.md]: I guess you meant `ODF's` --> `PDF's`

File: Makefile (hunk 1 above)
Comment (mmattel) [reply Makefile]: ah, just seen that it is defined in package.json/scripts

**Comment by mmattel** [comment]
LGTM, only a view fixes necessary

File: Makefile (hunk 1 above)
Comment (settermjd) [reply Makefile]: What's `yarn prose`?

File: docs/build-the-docs.md (hunk 2 above)
Comment (settermjd) [reply docs/build-the-docs.md]: I'm guessing so, as nothing generates ODF files.

File: Makefile (hunk 1 above)
Comment (settermjd) [reply Makefile]: Just looked at other files and see what this is about.

**Comment by tboerger** [comment]
I guess ready for merge.

**Pull request merged**